components/aws/sagemaker/tests/unit_tests/tests/common/dummy_component.py

```diff
@@ -0,0 +1,13 @@
+from tests.unit_tests.tests.common.dummy_spec import (
```

Comment:
Why do we need this dummy component? (I might have missed a comment about it if there is one)

Reply:
It's basically a mock. We just want to use the structure.